packages/esm-billing-app/src/helpers/functions.ts

```diff
@@ -34,7 +35,7 @@ export function calculateTotalAmount(lineItems: Array<LineItem>) {
 export const convertToCurrency = (amountToConvert: number) => {
   const formatter = new Intl.NumberFormat('en-KE', {
```

Comment:
Thanks @CynthiaKamau we should also configure the locale also here. Am not sure but can we explore using `i18next` to support it. If not possible it would be better to support both `locale` and `currency` to have the correct output.
```suggestion
  const formatter = new Intl.NumberFormat(config.locale, {
```